docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+mantl base directory /library/lvg.py
+
+
+Do all the computers to this point.
+
+## Creating Your Inventory
+
+Here is an example inventory file. It should be place in the root of the mantl directory.
+
+    [role=control]
+    control01 private_ipv4=172.16.222.6 ansible_ssh_host=172.16.222.6
+    control02 private_ipv4=172.16.222.7 ansible_ssh_host=172.16.222.7
+    control03 private_ipv4=172.16.222.8 ansible_ssh_host=172.16.222.8
+
+    [role=control:vars]
+    consul_is_server=true
+    lvm_physical_device=/dev/sda3
+
+    [role=worker]
+    resource01 private_ipv4=172.16.222.11 ansible_ssh_host=172.16.222.11
+    resource02 private_ipv4=172.16.222.12 ansible_ssh_host=172.16.222.12
+    resource03 private_ipv4=172.16.222.13 ansible_ssh_host=172.16.222.13
+
+    [role=worker:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [role=edge]
+    edge01 private_ipv4=172.16.222.16 ansible_ssh_host=172.16.222.16
+    edge02 private_ipv4=172.16.222.17 ansible_ssh_host=172.16.222.17
+
+    [role=edge:vars]
+    consul_is_server=false
+    lvm_physical_device=/dev/sda3
+
+    [dc=dc1]
+    control01
+    control02
+    control03
+    resource01
+    resource02
+    resource03
+    edge01
+    edge02
+
+I had to add the ansible_ssh_host line to run `playbooks/reboot-hosts.yml`
+
+The private_ipv4 is needed by several roles.
+
+The dc=dc1 group is needed to set `consul_dc_group` in the consul roles. And is specifically usd in this setup in the
+dnsmasq role.
+
+Note that dc1 is the default.  If you change the name of the data center in your inventory file
+you will need to set consul_dc in an environment variable.  For example, if you called your dc 'mydc' then you
+would need to enter:
+
+    ansible-playbook -u centos -i inventory -e consul_dc=mydc \
+            -e provider=bare-metal  -e @security.yml  terraform.sample.yml >& bare-metal.log
+
+the rest of the options will be discussed below.
+
+## Getting Started with Ansible
+
+
+Add your key to all the machines in your inventory
+
+    ansible all -i inventory  -u centos -k -m authorized_key -a "user=centos key=https://github.com/youraccount.keys"
+
+Note this makes use of your public key on github.  if you don't have a git hub account or a key pair on your git hub
+account, get one.
+
+the -k is needed cause I have to have it ask for a password at this point.
+
+now all commands can happen with out the password and -k option. Test with:
+
+    ansible all -i inventory -u centos -m ping
+
+You should get back a pong from each machine in your inventory.
+
+### Copy the /etc/host file over
+
+Add the nodes to /etc/hosts
+
+content is:
+
+        127.0.0.1 localhost localhost.localdomain localhost4 localhost4.localdomain4
+        ::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+        172.16.222.5 MyMac
+        172.16.222.6 control01
+        172.16.222.7 control02
+        172.16.222.8 control03
+        172.16.222.11 resource01
```

Comment:
The terraform files call these nodes workers, and we might want to keep that nomenclature consistent. Terraform also adds a dash, like `worker-01`.